rpm/scripts/preun.sh

```diff
@@ -0,0 +1,5 @@
+if [ "$1" = 0 ]; then
+  /sbin/service mad stop > /dev/null 2>&1
+  /sbin/chkconfig --del mad
+fi
+exit 0
```

Comment:
We don't want to fail pre-uninstall if we can't delete the service profile (and rollback the removal transaction).  Same with the post-uninstall.